lib/src/ui/widgets/hypertext.dart

```diff
@@ -46,9 +48,10 @@ class _AppHypertextState extends State<AppHypertext> {
       onExit: isEnabled ? (event) => onHover(false) : null,
       cursor: isEnabled ? SystemMouseCursors.click : SystemMouseCursors.basic,
       child: GestureDetector(
-        onTap: widget.onPressed,
+        onTap: widget.onTap,
+        onTertiaryTapUp: (details) => widget.onTertiaryTapUp?.call(),
```

Comment:
This could also be a convention or even better, a linter: to use underscore ('_') when it is not used? 'details'?

Reply:
Yes sure!